src/App.js

```diff
@@ -1,23 +1,9 @@
-import logo from './logo.svg';
-import './App.css';
+import HeroComponent from './components/HeroComponent';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div >
+      <HeroComponent/>
```

Comment:
can you please remove this since it wil cause merge conflicts

Reply:
removed 👍